### ISSUE TITLE
Env command usage

### ISSUE THREAD

**Issue by robeberhardt** [oursiberia/ssmenv#7]
help says:
```
USAGE
  $ ssmenv env STAGE
```

but I get:
```
$ ssmenv env dev
Error: Cannot find module 'tslib'
    at Object.<anonymous> (~/.nvm/versions/node/v9.9.0/lib/node_modules/ssmenv/lib/commands/env/index.js:3:17)
```

if I use: 
```
$ ssmenv env:dotenv dev
```

I get: 
```
THING2=blue 0 [ 'THING2=blue', 'THING1=5', 'BLARG=true' ]
THING1=5 1 [ 'THING2=blue', 'THING1=5', 'BLARG=true' ]
BLARG=true 2 [ 'THING2=blue', 'THING1=5', 'BLARG=true' ]
```

**Comment by bryanjswift** [comment]
I think this is two separate issues:

1. The `env` command isn't properly aliasing to `env:dotenv`
2. The output of `env:dotenv` includes the full array of arguments on each line.

The second is because of a change to an upstream dependency and the way I was doing output, ~will fix~ was fixed in [v0.2.0](https://github.com/oursiberia/ssmenv/releases/tag/v0.2.0). The first is more troubling and I'll have to investigate a bit.

**Comment by robeberhardt** [comment]
same thing is happening for me with `var` --

`env var staging color red` gives me that same error (cannot find module tslib)

`ssmenv var:set staging color red` works as expected

**Comment by bryanjswift** [comment]
Usage errors should be fixed in [v0.3.0](https://github.com/oursiberia/ssmenv/releases/tag/v0.3.0)
